main.py

```diff
@@ -258,8 +258,10 @@
                  'label': 'Blood Frenzy / Savage Combat',
                  'value': 'blood_frenzy'
              },
+             {'label': "Curse of Elements / Earth and Moon / Ebon Plaguebringer",
+              'value': 'curse_of_elements'},
```

Comment:
for a multiline dict like this, you should format it like the dict above
```suggestion
             {
                 'label': "Curse of Elements / Earth and Moon / Ebon Plaguebringer",
                 'value': 'curse_of_elements'
             },
```